modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2597,6 +2597,40 @@ TEST_P(Test_ONNX_layers, where_node)
     testONNXModels("where_layer");
 }
 
+TEST_P(Test_ONNX_layers, Conformance_Gemm_all_attributes) {
```

Comment:
> Conformance

That confuses in test logs.
Avoid using `conformance` wording outside of `test_onnx_conformance.cpp`

Reply:
Fixing via https://github.com/opencv/opencv/pull/24315